.github/workflows/publish.yaml

```
@@ -0,0 +1,23 @@
+name: Publish to PyPI
+on:
+  push:
+    tags:
+      - "*.*.*"
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Is there a reason not to use v2?
```suggestion
      - uses: actions/checkout@v2
```

---

Thanks. I'll fix this.